### PR TITLE
377-aws-alarm-for-disk-space-on-jenkins

### PR DESCRIPTION
**Pull request by benvand**
* Add permissions for jenkins instance to PutMetricData in to Cloudwatch (used with cloudwatch-logs-agent)
* Add new alarms for disk space on jenkins

https://trello.com/c/PdXsLT6d/377-aws-alarm-for-disk-space-on-jenkins

### CODE REVIEW

File: terraform/accounts/main/alarms.tf [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  comparison_operator = "LessThanOrEqualToThreshold"
+  threshold           = "10000000000"
+
+  // Email slack
+  alarm_actions = ["${module.alarm_email_sns.email_topic_arn}"]
+  ok_actions    = ["${module.alarm_recovery_email_sns.email_topic_arn}"]
+}
+
+resource "aws_cloudwatch_metric_alarm" "jenkins_main_volume_disk_space" {
+  alarm_name        = "jenkins-data-volume-disk-space"
```
Comment (katstevens) [inline terraform/accounts/main/alarms.tf]: Should this be a different name? `jenkins-main-volume-disk-space`?

File: terraform/accounts/main/jenkins_instance_profile.tf [hunk 2]
```diff
@@ -159,9 +159,28 @@ data "aws_iam_policy_document" "cloudwatch-policy" {
...
+}
+
+data "aws_iam_policy_document" "cloudwatch-metrics-policy" {
+  statement {
+    actions = [
+      "cloudwatch:PutMetricData",
+    ]
+
+    resources = [
+      "*",
```
Comment (katstevens) [inline terraform/accounts/main/jenkins_instance_profile.tf]: Does this mean 'everything'? Or are we waiting on a hardcoded value?

Comment (benvand) [reply terraform/accounts/main/jenkins_instance_profile.tf]: Means everything, and according to the AWS interface has to be everything.

File: terraform/accounts/main/alarms.tf (hunk 1 above)
Comment (benvand) [reply terraform/accounts/main/alarms.tf]: Nice catch 👍 updated

**Pull request merged**